src/Nethermind/Nethermind.JsonRpc/Modules/Eth/EthRpcModule.cs

```diff
@@ -285,7 +285,7 @@ public ResultWrapper<byte[]> eth_sign(Address addressData, byte[] message)
         }
 
         if (_logger.IsTrace) _logger.Trace($"eth_sign request {addressData}, {message}, result: {sig}");
-        return ResultWrapper<byte[]>.Success(sig.Bytes);
+        return ResultWrapper<byte[]>.Success(sig.Bytes.ToArray());
```

Comment:
Can we either get Memory<byte> to the signature, or just simpler add JSON converter for whole signature object?

Reply:
Switched to Memory